content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

```diff
@@ -46,90 +46,91 @@ Enter the name you want to use for the administrator account.
 
 Enter and confirm the admin user passwords.
 
-Select the **root**, **builtin_administrators** and **builtin_users** groups from the **Auxiliary Group** dropdown list.
+Select the **builtin_administrators** group from the **Auxiliary Group** dropdown list.
 
 {{< trueimage src="/images/SCALE/22.12/AddingAdminUserAuxiliaryGroup.png" alt="Add Admin User to builtin_administrators" id="2: Add Admin User to builtin_administrators" >}}
 
-Click **Save**.
-
-Log out of the TrueNAS system and then log back in using the admin user credentials. Once you are back in the TrueNAS web UI, determine that the admin user credentials are working properly with your network configuration.
-
-### Configuring the Admin User Account for Nextcloud App
-The Nextcloud application, configured on the **Apps > Available Applications** screen, requires including sudo permissions for the administrator account. 
-To verify, or manage the local administrator account, go to **Credentials > Local Users** and click on the administrator user row to expand it, then click **Edit** to open the **Edit User** configuration screen.
-
 Scroll down to the **Authentication** settings and select the proper **Allow sudo** authorization settings.
+Some applications, such as [Nextcloud]({{< relref "InstallNextCloudMedia.md" >}}), require sudo permissions for the administrator account.
+By default, TrueNAS SCALE sets admin account authorization to **Allow all sudo commands**.
```

Comment:
No it does not automatically set this for the admin unless the admin is created during the iso installation process. We need to be careful about saying this as a default because someone upgrading to SCALE Bluefin or migrating from CORE does not necessarily have the admin account created for them and they must choose the sudo settings.
Also, it is select the option not click the checkbox.

Reply:
Ok, but I think we should somehow still communicate that this is the automated permissions default. What about:

```suggestion
For administrator accounts generated during the initial installation process, TrueNAS SCALE sets authorization to **Allow all sudo commands**.
```